roles/matrix-bridge-appservice-slack/tasks/setup_install.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+# If the matrix-synapse role is not used, `matrix_synapse_role_executed` won't exist.
+# We don't want to fail in such cases.
+- name: Fail if matrix-synapse role already executed
+  fail:
+    msg: >-
+      The matrix-bridge-appservice-slack role needs to execute before the matrix-synapse role.
+  when: "matrix_synapse_role_executed|default(False)"
+
+- name: Ensure Appservice Slack image is pulled
+  docker_image:
+    name: "{{ matrix_appservice_slack_docker_image }}"
+    source: "{{ 'pull' if ansible_version.major > 2 or ansible_version.minor > 7 else omit }}"
+
+- name: Ensure Appservice Slack paths exist
+  file:
+    path: "{{ matrix_appservice_slack_base_path }}"
+    state: directory
+    mode: 0750
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_username }}"
+
+- name: Check if an old database file already exists
+  stat:
+    path: "{{ matrix_appservice_slack_base_path }}/user-store.db"
+
+- name: Ensure Matrix Appservice Slack config installed
+  copy:
+    content: "{{ matrix_appservice_slack_configuration|to_nice_yaml }}"
+    dest: "{{ matrix_appservice_slack_base_path }}/config.yaml"
+    mode: 0644
+    owner: "{{ matrix_user_username }}"
+    group: "{{ matrix_user_username }}"
+
+- name: Ensure appservice-slack registration.yaml installed
+  copy:
+    content: "{{ matrix_appservice_slack_registration|to_nice_yaml }}"
+    dest: "{{ matrix_appservice_slack_base_path }}/slack-registration.yaml"
```

Comment:
This should use the new `matrix_appservice_slack_config_path` variable that I've mentioned above.

And instead of calling the file `slack-registration.yaml`, it can just call it `registration.yaml`.
The file is nested multiple levels into a Slack bridge directory, so it's obviously a Slack bridge registration file, without having to explicitly name it so.